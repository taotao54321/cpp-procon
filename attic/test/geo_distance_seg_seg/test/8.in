50
0 0 10 0 33 43 62 29
0 0 10 0 0 8 52 56
0 0 10 0 56 19 11 51
0 0 10 0 43 5 8 93
0 0 10 0 30 66 69 32
0 0 10 0 17 47 72 68
0 0 10 0 80 23 49 92
0 0 10 0 64 69 51 27
0 0 10 0 90 24 35 20
0 0 10 0 44 10 62 84
0 0 10 0 63 1 10 36
0 0 10 0 76 31 29 97
0 0 10 0 75 91 90 44
0 0 10 0 34 25 29 30
0 0 10 0 27 26 43 34
0 0 10 0 4 60 49 20
0 0 10 0 56 32 72 13
0 0 10 0 90 9 19 5
0 0 10 0 95 49 27 19
0 0 10 0 97 24 96 49
0 0 10 0 56 84 93 45
0 0 10 0 7 6 9 54
0 0 10 0 52 65 83 38
0 0 10 0 1 90 30 37
0 0 10 0 95 56 63 11
0 0 10 0 27 42 6 68
0 0 10 0 12 1 10 80
0 0 10 0 58 71 31 14
0 0 10 0 47 64 97 25
0 0 10 0 38 31 18 87
0 0 10 0 51 87 13 79
0 0 10 0 95 50 50 13
0 0 10 0 62 34 73 47
0 0 10 0 21 30 40 57
0 0 10 0 78 26 3 97
0 0 10 0 8 93 88 38
0 0 10 0 85 93 88 20
0 0 10 0 11 46 87 10
0 0 10 0 9 87 68 3
0 0 10 0 73 0 74 5
0 0 10 0 83 52 70 87
0 0 10 0 2 10 5 10
0 0 10 0 0 96 42 85
0 0 10 0 60 47 24 31
0 0 10 0 58 86 19 0
0 0 10 0 15 55 82 74
0 0 10 0 61 6 2 68
0 0 10 0 65 70 18 23
0 0 10 0 89 6 19 30
0 0 10 0 55 32 93 74
