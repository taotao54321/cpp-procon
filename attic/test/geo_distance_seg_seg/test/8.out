48.7647413609
8.0000000000
42.1422453394
32.5115501316
62.8917041826
47.5184174821
73.2775519209
49.0917508345
32.0156211872
35.4400902933
30.0407259511
71.7435657265
91.3016976841
34.6554469023
31.0644491340
40.8583760880
56.0357029045
10.2956301410
25.4950975680
90.2496537389
91.1852064126
6.0000000000
76.6000950144
42.0594816896
54.1294744109
45.3100430368
2.2360679775
25.2388589282
73.2203241368
41.7731971484
79.0569415042
42.0594816896
62.1288982680
31.9530906173
65.6297537633
75.5028413048
80.5232885568
42.0000269338
49.1866087739
63.0000000000
89.6270048590
10.0000000000
90.3341395261
34.0147027034
9.0000000000
55.2268050859
41.0813826447
24.3515913238
31.2972972973
55.2177507691
